app/components/projetos/index.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <section
+    class="grid border-t border-gray-300 dark:border-t dark:border-gray-700 md:grid-cols-2 py-8 px-4 md:py-11 md:px-24 text-gray-700 dark:bg-gray-800"
+  >
+    <div class="flex items-center justify-center mx-auto py-[80px]">
+      <h2 class="dark:text-white text-5xl md:text-5xl font-bold m-0 px-4">
+        projetos
+      </h2>
+    </div>
+
+    <div class="dark:text-white px-4">
+      <div class="mb-3 px-1">
+        <h3 class="font-bold text-lg">GC Manager</h3>
+        <p class="text-gray-700 font-semibold dark:text-white">
+          Sistema para gerenciamento de pequenos grupos.
+        </p>
+        <p>Em construção</p>
+        <p>
+          <a
+            href="https://github.com/dinobergviana/gc-manager"
+            class="mb-4 text-blue-600 hover:text-sky-500 dark:text-yellow-500 dark:hover:text-yellow-600 ease-in"
+            target="_blank"
+            >Repositório</a
+          >
+          <span class="text-gray-700"> - </span>
```

Comment:
@Diobegue se tu observar la no site que ta no ar, o separador aqui é do tipo `bullet`. Testa substituir o traço por esse código. Do jeito que ta aqui. Esse código no fim das contas vai virar o pontinho la.
```suggestion
          <span class="text-gray-700"> &bull; </span>
```
Caso tu queira entender mais sobre simbolos html, dá uma olhada nessa documentação:
https://www.w3schools.com/html/html_symbols.asp